HPXMLtoOpenStudio/resources/data/default_panels.csv

```diff
@@ -0,0 +1,31 @@
+Load Name,Voltage,Power Rating,Units,Breaker Spaces,Load Type,Notes
+lighting,120,3,W/sqft,0,lighting,Multiplier for conditioned floor area to account  for capacity impacts of general lighting.
```

Comment:
Can we change 0 space to 2? I have it in future work to dynamically determine this based on sqft or the number of room types. I think people may get confused by 0 space here.

Reply:
Changed in https://github.com/NREL/OpenStudio-HPXML/pull/1946/commits/1b7843050a19595179ae27f1cb6134f48650c06d.